### DISCUSSION TITLE
Can't install supermaven or continue extensions

### DISCUSSION THREAD

**Discussion by AverageGuy** [microsoft/vscode-discussions#1823] [Extension Development QnA] [answered]
Technically this is incorrect.  I installed both but when I open them they just sit there with a progress bar at the top of the page running across forever.  In the case of supemaven I got messages like Error: canceled.  The install didn't give me any errors just when I tried to configure/use them.  If this is the wrong place to bring up these issues, please direct me to the proper location.
Thanks,
Jim.

**Comment by gjsjohnmurray** [comment]
> If this is the wrong place to bring up these issues, please direct me to the proper location.

Please start by reading https://github.com/microsoft/vscode-discussions/discussions/5

**Comment by AverageGuy** [comment]
I am guessing what you are referring to is this statement "Discussions focus only on extension development to learn, experiment, and grow sustainably."  So tell me where to actually get help with usage since this is only for developers.

**Comment by gjsjohnmurray** [comment]
Several suggestions on #5 in the section headed "Other places to participate with VS Code:"

**Comment by alefragnani** [comment, marked as answer]
Hi @AverageGuy ,

If you are having issues while using some specific extension, the best place to look for info/help would be with the extension’s author, either its homepage or issue tracker. You will find both informations (if provided) in the extension’s marketplace page. If the extension’s author says the issue is related to VS Code itself, then you should report the issue in VS Code repository (https://github.com/microsoft/vscode/issues).

I did a quick search in the Marketplace, and if the extensions you commented above are the first hit, I guess Supermaven doesn’t have a public issue tracker but instead, an e-mail support. Continue extension, on the other hand, seems to have an issue repo to look for (https://github.com/continuedev/continue/issues).

I would suggest you to report your issue on their channel. They would probably be able to help you with your issue.

Hope this helps.

**Reply by AverageGuy** [reply]
Hi aelfragnani,
Thank you for the help.  I'll check out the suggestions.
Jim.
